### DISCUSSION TITLE
How to set or edit the namespace for the Operator deployment (make deploy)

### DISCUSSION THREAD

**Discussion by stevelaclasse** [kubernetes-sigs/kubebuilder#2239] [Support] [answered]
Hello,

I am testing an operator on a local Kubernetes cluster and saw that that the Operator is deployed in his default namespace "controller-manager-system" . I saw some yaml files where this namespace could be obtained. 

My question is how to set the namespace where the operator will be deployed. 

thanks.

**Comment by camilamacedo86** [comment, marked as answer]
Hi @stevelaclasse,

By using the Makefile targets command when you run `make deploy IMG=image` ([More info](https://book.kubebuilder.io/quick-start.html#run-it-on-the-cluster)) a namespace with the project name concat with "-system" will be created and your project will be applied there.  

See that it is defined in the config//kustomization.yaml file: https://github.com/kubernetes-sigs/kubebuilder/blob/v3.5.0/testdata/project-v3/config/default/kustomization.yaml#L1-L9

Also, check here the manifest to create the namespace: https://github.com/kubernetes-sigs/kubebuilder/blob/26d12ab1134964dbbc3f68877ebe9cf6314e926a/testdata/project-v3/config/manager/manager.yaml#L1-L6

Therefore, to set the namespace where the operator will be deployed you can check the kustomize configuration for example.

**Comment by camilamacedo86** [comment]
Closing since it is answered.
